packages/synthetix-cli/tasks/interact.js

```diff
@@ -0,0 +1,18 @@
+const { task } = require('hardhat/config');
+
+const { TASK_INTERACT } = require('../task-names');
+
+task(TASK_INTERACT, 'Interacts with a given Synthetix deployment')
```

Comment:
Description should read "Interacts with a given modular system deployment" (it's not just Synthetix now)

Also, I think it's safe to remove all params for now. Most of them don't apply anymore in the v3 architecture, and we should probably add them only as they are actually used.